applications/decentralized_oracles.md

```diff
@@ -0,0 +1,135 @@
+We plan to replace the scheduled jobs with phat contracts to improve decentralization and demonstrate the interoperability between ink! smart contracts (Shiden/Astar) and ink! phat contracts (Khala/Phala).
+
+The dApp is already live on Shiden.
+https://portal.astar.network/shiden/dapp-staking/dapp?dapp=x6ykus6l6ch4eozitzsyjscxh2agk2ky9g6a2xeu1w9fftp
+
+Website:  https://lucky.substrate.fi/  
+
+Documentation: https://polkaverse.com/11201/introducing-the-d-app-lucky-37998  
+
+The dApp was awarded in the Polkadot Hackathon Europe Edition, in the category ink! Smart Contracts.  
+
+### Team Code Repos
+
+GitHub accounts of team members:
+- https://github.com/GuiGou12358
+- https://github.com/arnobase
+
+GitHub for the smart contracts used in the dApp Lucky:
+ - https://github.com/GuiGou12358/lucky-contracts
+
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+
+- **Total Estimated Duration:** 2 months
+- **Full-Time Equivalent (FTE):**  1 FTE
+- **Total Costs:** 16,000 USD
+
+### Milestone 1  — VRF
+
+- **Estimated duration:** 1 month
+- **FTE:**  1 FTE 
+- **Costs:** 8,000 USD
+
+|  Number | Deliverable                                   | Specification                                                                                                                                                                                                                                 |
+|--------:|-----------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **0a.** | License                                       | Apache 2.0                                                                                                                                                                                                                                    |
+| **0b.** | Documentation                                 | We will provide documentation that explains how to use these oracles and test them                                                                                                                                                            |
+| **0c.** | Testing and Testing Guide                     | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests.                                                                               |
+|      1. | Phat contract: VRF Oracle                     | We will deliver the phala offchain rollup to compute the VRF                                                                                                                                                                                  |
```

Comment:
Would be nice if you can give out an sample API call in ink. It's not necessary to be a finalized definition, but a sample call will provide the reader better understanding of how it can be used once implemented. Same for the Graph API oracle.